pkg/envconf/config.go

```diff
@@ -44,6 +44,7 @@ type Config struct {
 	dryRun                  bool
 	failFast                bool
 	disableGracefulTeardown bool
+	kubeContext             string
```

Comment:
@vladimirvivien and @harshanarayana, even if we're parsing directly from flags in `ResolveClusterContext()`, I thought about adding the flag to the `Config` object for consistency.

If you think it's not necessary, I'm happy to drop this change